osu.Game/Database/IPresentImports.cs

```diff
@@ -12,6 +12,6 @@ public interface IPresentImports<TModel>
         /// <summary>
         /// Fired when the user requests to view the resulting import.
         /// </summary>
-        public Action<IEnumerable<TModel>> PresentImport { set; }
+        public Action<IEnumerable<ILive<TModel>>> PresentImport { set; }
```

Comment:
Rider is claiming this interface can be made covariant but I'm not sure whether that does anything. Generic variance is one of these things that I'm vaguely aware of but can never grasp fully due to its abstractness...